proto/tendermint/farming/v1beta1/genesis.proto

```diff
@@ -103,6 +98,25 @@ message QueuedStakingRecord {
   QueuedStaking queued_staking = 3 [(gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"queued_staking\""];
 }
 
+// TotalStakingRecord is used for import/export via genesis json.
+message TotalStakingRecord {
```

Comment:
If I am understanding your comment @hallazzang, it is about using singular or plural, right? I am voting for plural `TotalStakingsRecord` since we use plural in `TotalStakings` and rest endpoint `total_stakings`.

Reply:
Sure I just rename to `TotalStakingsRecord` due to endpoint dependency with `total_stakings`